Hurrican/src/Console.cpp

```diff
@@ -422,8 +422,16 @@ void ConsoleClass::CheckCommands() {
         DirectGraphics.ShowBackBuffer();
 
         // Screenshot machen
-        DirectGraphics.TakeScreenshot("MiniMap", RENDERWIDTH, RENDERHEIGHT);
-        this->print("Mini Map saved !");
+        bool success = DirectGraphics.TakeScreenshot("MiniMap.bmp", RENDERWIDTH, RENDERHEIGHT);
+
+        // Show the map for 2 seconds
+        Timer.wait(2000);
```

Comment:
Shouldn't this be inside the success branch?

Reply:
```success``` is only there for the status of the file being saved.
The delay is there to keep the map shown on screen for 2 seconds.
(The minimap gets drawn on screen and a screenshot is saved).
Even when the map failed to save, you can still see it on screen.